resources/js/control_block.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function () {
+    $.ajaxSetup({
+        headers: {
+            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        }
+    });
+
+    $('.add-friend').click(function () {
+        var username = $(this).data('friend-name');
+
+        var url = "/" + username + "/addFriend";
```

Comment:
```suggestion
        var url = '/' + username + '/addFriend';
```
mà url thì ngta thường để dạng add-friend nhé